_partials/_hypershift-intro.md

```diff
@@ -0,0 +1,13 @@
+You can use hypershift to migrate existing PostgreSQL databases in one step, and
+enable compression and hypertable creation on the fly.
+
+Hypershift can migrate your data in Timescale Cloud from these sources:
+
+*   Standard PostgreSQL databases
+*   Amazon RDS databases
+*   Other Timescale databases, including Managed Service for TimescaleDB, and
+    self-hosted TimescaleDB
+
+Because compression is enabled during the migration, you do not need to have the
+maximum amount of storage available in the target database before you start
+migration.
```

Comment:
```suggestion
Because compression is enabled during the migration, a maximum amount of storage is not required in the target database before you start the migration.
```